dome9CloudBots/bots/storage_account_enable_https_traffic_only.py

```diff
@@ -0,0 +1,34 @@
+# What it does: enable Storage account Secure transfer required field
+# Usage: AUTO: storage_account_enable_https_traffic_only
+# Limitations: None
+
+import logging
+from msrestazure.azure_exceptions import CloudError
+from azure.mgmt.storage import StorageManagementClient
+from azure.mgmt.storage.models import StorageAccountUpdateParameters
+
+
+def run_action(credentials, rule, entity, params):
+    logging.info(f'{__file__} - run_action started')
+    subscription_id = entity.get('accountNumber')
```

Comment:
Why you are using `get` here?
Just use `entity['accountNumber']` - and if it doesn't exists an exception will be raised.

And if you want to specify a specific message, just do:
```
try:
    subscription_id  = entity['accountNumber']
except KeyError:
    logging.info .... (BTW why not logging.error?)
```